commitlint.config.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  extends: ["@commitlint/config-conventional"],
+  rules: {
+    "scope-enum": [2, "always", ["osmInterface", "dependencies", "configurations"]],
```

Comment:
osmInterface is not a valid scope, 
think of the type of elements osm exposes: node, way, relation , changeset

Reply:
done